README.md

```diff
@@ -48,6 +48,7 @@ Should include:
 - `npm install` to install dependencies
 - `npm run cypress` to open the cypress testing window
 - `npm test` to run jest / node tests
+- `npm run test:bonus` to run jest / node tests **including the bonus tests**
```

Comment:
Nit: I'd suggest putting this as _italic_; it doesn't need to be significantly more emphasized than what's before it.

```suggestion
- `npm run test:bonus` to run jest / node tests _including the bonus tests_
```

_(I used to get confused or distracted when old assignments did this to me)_